### DISCUSSION TITLE
can i build booking time slot?

### DISCUSSION THREAD

**Discussion by maxsim86** [Budibase/budibase#2105] [Help]
Hi, can i buid a time slot calendar booking for appointment parent

**Comment by shogunpurple** [comment]
Hi @maxsim86 - would you be able to provide a little more detail about what exactly you are trying to build?

Is it an app that allows you to book time slots with calendar integration, something akin to calendly/calendso? Or are there other use cases that you had in mind?

If your app needs direct calendar integration, budibase does not have this functionality yet - we may implement something in future but it's not on our roadmap. 

Thanks!

**Reply by oswaldB** [reply]
@maxsim86 if you need you can still use the webhook from automation (or zapier or integromat) to connect with calendars

**Comment by msj1542** [comment]
Has there been any update to this functionality since this thread was created or does the status of a calendar/booking feature remain the same?

**Reply by shogunpurple** [reply]
You can work with dates etc in budibase, but there's no direct calendar integration. You would have to do this with the REST connector and the google calendar API.

**Reply by msj1542** [reply]
That would be fine, but is there a semi-straightforward way to include the UI portion of a calendar into Budibase as well through this process (or even just embed a web link within a page somewhere even, if this is easiest), or would what I am wanting require a custom coding it in order to build it from scratch?  (If the latter, would you happen to have or know of any relevant resources that you would care to point me in the right direction of?)  Thanks!
